components/CustomButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
Nice that you created a button component in it's own file like this anbd then importing it where you need it ⭐️